spec/findSpec.js

```diff
@@ -632,6 +645,36 @@ describe('find', (it) => {
         next: res.next
       });
 
+      t.is(res.results.length, 2);
+      t.is(res.results[0].counter, 2);
+      t.is(res.results[1].counter, 1);
+      t.true(res.hasPrevious);
+      t.false(res.hasNext);
+    });
+  });
+  it.describe('test with date as paginated field using dot notation', (it) => {
+    it('should query first few pages', async (t) => {
+      const collection = t.context.db.test_paging_date_in_object;
+      // First page of 2
+      var res = await paging.find(collection, {
```

Comment:
just to verify - this runs all the new codepaths (both next and previous)?

Reply:
No, I'll add a previous test.